docs/wiki/Installation.md

```
@@ -85,6 +85,19 @@ says so with the install line for the one you named:
 	HINT: pip install "django-aiogram[redis]"
 ```
 
+**Two extras are not transports.** `hiredis` swaps redis-py's parser for the C one, and
+`prometheus` (`prometheus-client>=0.20`) installs the client the shipped exporter fills:
+
+```shell
```

markdownlint-cli2 (0.23.2)

[warning] Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)